architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -34,6 +37,44 @@ you are responsible for updating it to reflect requirements of newer versions of
 the {product-title} master.
 ====
 
+=== General Admission Rules
+Starting in 1.3, OpenShift uses a single admission chain for Kubernetes and OpenShift
```

Comment:
Starting in 
ifdef::openshift-enterprise[]
3.3,
endif::[]
ifdef::openshift-origin[]
1.3,
endif::[]
{product-title} uses . . .